### ISSUE TITLE
//move not working properly

### ISSUE THREAD

**Issue by ghost** [SBPrime/AsyncWorldEdit-Premium#88]
If //move is used and the resulting blocks placed are not 100% outside the WorldEdit selection, the blocks will not be placed in their original layout.
![2016-07-24_18 56 26](https://cloud.githubusercontent.com/assets/2307465/17087109/5bc73d0e-51d0-11e6-9766-b00ea29b720a.png)

**Comment by SBPrime** [comment]
Quote from forums:
The //move command in the latest awe command is severvely bugged, so that it destroys structures.
This happens under worldedit 6.1.3 and also with the latest dev version (have tested it with 6.1.3)

For the pastebin, I have turned on debug mode in worldedit and awe, and then started a server, and then done on 2
differnt areas //move, and then //undo

http://pastebin.com/qP9Vc24A (the procedure below with the image is what you see in the log)
http://pastebin.com/i5mSNmC1 awe config file (should be default)
http://pastebin.com/dKPZ2v80 worldedit config

First area:
http://i.imgur.com/DzAKB4w.png
structure (vaulting) after //move command - everything fine

http://i.imgur.com/w5CSRKr.png
structure after //undo command - destroyed

Second area
http://i.imgur.com/wag1WuP.png
structure after //move - parts of the diorite are missing.

After doing undo here it turns without error to its initial state.

So to sum it up, in the example of the first area move works fine and undo destroys the structure,
and in the second area move destroys the structure and undo works fine.

**Comment by SBPrime** [comment]
Fixed in v3.4.1
